src/main/java/duke/command/CommandNotFound.java

```diff
@@ -0,0 +1,27 @@
+package duke.command;
+
+import duke.Storage;
+import duke.TaskList;
+import duke.Ui;
+
+/**
+ * CommandNotFound - when user enters a Command and Duke
+ *         does not understand the command.
+ */
+public class CommandNotFound extends Command {

```

Comment:
Perhaps consider changing the name of this class to be InvalidCommand or UnknownCommand, to be consistent with the naming of the rest of the command classes.